.github/workflows/verify.yml

```diff
@@ -90,10 +105,15 @@ jobs:
           bundle exec standardrb
 
       - name: Install Node packages
-        if: steps.package_json_check.outputs.files_exists == 'true'
+        if: steps.package_json_check.outputs.files_exists == 'true' && steps.bun_lock_check.outputs.files_exists == 'false'
         run: |
           yarn install --pure-lockfile
 
+      - name: Install Bun packages
+        if: steps.package_json_check.outputs.files_exists == 'true' && steps.bun_lock_check.outputs.files_exists == 'true'
+        run: |
+          bun install --pure-lockfile
```

Comment:
Is `--pure-lockfile` supported? I can't find it mentioned in the docs. (`bun` doesn't seem to complain if you pass it invalid arguments).

Reply:
Yes, it's in the docs for `bun install` https://bun.sh/docs/cli/install.

Here's a direct link to the docs for this option.

https://github.com/oven-sh/bun/blob/bbc70d2b73c80630a48cab5a24d8e00b07032e12/docs/install/index.md?plain=1#L52-L55

Reply:
Looks like it's `--frozen-lockfile` rather than `--pure-lockfile`?